src/modules/shared/components/MediaCardList/MediaCardList.tsx

```diff
@@ -137,6 +137,7 @@ const MediaCardList: FC<MediaCardListProps> = ({
 	return (
 		<div
 			className={clsx(
+				'c-media-card-list',
```

Comment:
Het lijkt me niet de bedoeling om global styles te gebruiken in een component. Ik zou hier css modules gebruiken of de type extenden met `DefaultComponentProps` en dan zo de globale class meegeven.